### ISSUE TITLE
Flash electron data has some weird values

### ISSUE THREAD

**Issue by zain-sohail** [OpenCOMPES/sed#464]
#459 after discussion, I was looking at the raw data and in the electron channels, there are some pulseIds that shouldn't exist. The h5 file contains them so it's not an sed issue but should be figured out and address in the loading step.
This is from the tutorial dataset `Gd_W110` filename `FLASH1_USER3_stream_2_run44799_file1_20230323T005636.1`.h5
![image](https://github.com/OpenCOMPES/sed/assets/10053031/3a5b4406-fee4-4603-b570-7898f741945f)

**Comment by rettigl** [comment]
These could maybe just be dark counts? How is the DLD triggered, how many microbunch triggers are being sent in-between train resets? You can filter them out during loading, but I suppose this is not necessary, it anyways should not be related to #459

**Comment by zain-sohail** [comment]
>  it anyways should not be related to #459

Yes, I didn't mean that they were related, just that the discussion made me look at the raw data in more detail.


> These could maybe just be dark counts? How is the DLD triggered, how many microbunch triggers are being sent in-between train resets? You can filter them out during loading, but I suppose this is not necessary,

But yes that makes sense. I'd ping @kutnyakhov for this.
